### PR TITLE
Fix raisecom fan speed sensor limits

### PR DESCRIPTION
**Pull request by vitalisator**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/raisecom.yaml [hunk 1]
```diff
@@ -18,6 +18,8 @@ modules:
                     num_oid: '.1.3.6.1.4.1.8886.1.1.5.2.2.1.2.{{ $index }}'
                     snmp_flags: ['-OeQUs', '-Pu'] # workaround for underscores in mib
                     descr: 'Fan {{ $index }} Speed'
+                    low_limit: 0
```
Comment (PipoCanaja) [inline includes/definitions/discovery/raisecom.yaml]: Are you sure that a limit of 0 is good idea ? it probably means that the fan is not running at all ... 
I would put something like 100, at least to ensure the fan is not stopped  ?

Comment (PipoCanaja) [reply includes/definitions/discovery/raisecom.yaml]: And most importantly, have you checked that the MIB does not provide any values for those limits (that would be quite unusual to provide fan min/max but Xmas is coming, let's dream :) )

Comment (vitalisator) [reply includes/definitions/discovery/raisecom.yaml]: yes, sure :-) the raisecom os has different fan speed level:
SpdLevelIndex   FanDueSpeed(r/min)  TemperScale         
--------------------------------------------------------
1               0                   47                  
2               4400                57                  
3               7700                67                  
4               9350                90                  

so 0 rpm is a legitime value if the temperature is below 47°C.
Unfortunately that is not documented in mib, but one can see that in cli and docs.

The Fan failures are detected via Fan state sensor.

Comment (PipoCanaja) [reply includes/definitions/discovery/raisecom.yaml]: OK.

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-59-release-changelog-december-2019/10558/1

**Pull request merged**